README.md

```diff
@@ -24,6 +24,374 @@ its own version allows changes to be made to the client outside of updates to th
+**strongly** recommend choosing the version of this project to use from the latest [Conjur OSS 
+suite release](https://docs.conjur.org/Latest/en/Content/Overview/Conjur-OSS-Suite-Overview.html). 
+Conjur maintainers perform additional testing on the suite release versions to ensure 
+compatibility. When possible, upgrade your Conjur version to match the 
+[latest suite release](https://docs.conjur.org/Latest/en/Content/ReleaseNotes/ConjurOSS-suite-RN.htm); 
+when using integrations, choose the latest suite release that matches your Conjur version. For any 
+questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
+
+### Setup
+The Conjur Java SDK can be imported manually through building the source code locally, 
+or by using a dependency configuration to import from Maven Central. Please refer to
+the following instructions for your specific use case.
+
+#### Using the Source Code
+
+You can grab the library's dependencies from the source by using Maven **or** locally
+by generating a JAR file and adding it to the project manually.
+
+To do so from the source using Maven, following the setup steps below:
+
+1. Create new Maven project using an IDE of your choice
+2. If you are using Maven to manage your project's dependencies, include the following
+   Conjur API dependency snippet in your `pom.xml` under `<project>`/`<dependencies>`:
+
+```xml
+    <dependency>
+      <groupId>com.cyberark</groupId>
+      <artifactId>conjur-sdk-java</artifactId>
+      <version>4.0.0</version>
+    </dependency>
+```
+
+_NOTE:_ Depending on what version of the Java compiler you have, you may need to update
+the version. At this time, the `{version}` that we are targeting compatibility with is
+Java 8:
+
+```xml
+  <properties>
+    <maven.compiler.source>{version}</maven.compiler.source>
+    <maven.compiler.target>{version}</maven.compiler.target>
+  </properties>
+```
+
+3. Run `mvn install -DskipTests` in this repo's directory to install Conjur API into your
+   local maven repository.
+
+#### Using the Jarfile
+
+If generating a JAR is preferred, you can build the library locally and add the dependency
+to the project manually by following the setup steps below:
+
+1. Clone the Conjur Java API repo locally: `git clone {repo}`
+2. Go into the cloned repository with `cd conjur-sdk-java`
+3. Run `mvn package -DskipTests` to generate a JAR file. The output `.jar` files will be located
+   in the `target` directory of the repo
+
+_NOTE:_ The above command runs `mvn package` without running the integration tests, since
+these require access to a Conjur instance. You can run the integration tests with
+the `./bin/test_integration` script. For more information on how to run the tests, take a look at
+our [Contributing](https://github.com/cyberark/conjur-api-java/blob/main/CONTRIBUTING.md) guide.
+
+4a. For Intellij, Follow the steps outlined [here](https://www.jetbrains.com/help/idea/library.html)
```

Comment:
4a, b and c are a single, unbroken line in the rendered MD: https://github.com/cyberark/conjur-sdk-java/tree/update-usage-documentation#using-the-jarfile

Also, the code blocks below are rendered as one continuous block, including backticks.